### ISSUE TITLE
Diff for all exercises shows different port in playwright.config.ts compared with solution files

### ISSUE THREAD

**Issue by dg1234uk** [epicweb-dev/full-stack-testing#14]
For all exercises the diff shows that `playwright.config.ts` differs from the solution files by one line

`- const PORT = process.env.PORT || '4000'`
`+ const PORT = process.env.PORT || '3000'`

It appears that the script `post-set-playground.js` changes the port in the problem files `playwright.config.ts` to 4000, however, it leaves the solution files `playwright.config.ts` port set at 3000 resulting in the diff.

**Comment by kentcdodds** [comment]
Yes. I need to find a good way to remove this from the diff. I have some ideas. Thanks for opening the issue!
